<?xml version="1.0" encoding="UTF-8"?><testrun name="ServerUnitTests" project="Record_Indexer" tests="10" started="10" failures="0" errors="1" ignored="0">
  <testsuite name="server.ServerUnitTests" time="106.145">
    <testcase name="validateUserTest" classname="server.ServerUnitTests" time="0.177"/>
    <testcase name="invalidUserTest" classname="server.ServerUnitTests" time="1.0"/>
    <testcase name="invalidInputUserTest" classname="server.ServerUnitTests" time="0.004"/>
    <testcase name="getProjectsTest" classname="server.ServerUnitTests" time="1.046"/>
    <testcase name="getProjectsInvalidTest" classname="server.ServerUnitTests" time="0.004"/>
    <testcase name="getSampleImageTest" classname="server.ServerUnitTests" time="2.072"/>
    <testcase name="downloadBatchTest" classname="server.ServerUnitTests" time="2.134"/>
    <testcase name="getFieldsFromProjectTest" classname="server.ServerUnitTests" time="2.057"/>
    <testcase name="getAllFieldsTest" classname="server.ServerUnitTests" time="1.053"/>
    <testcase name="submitBatchTest" classname="server.ServerUnitTests" time="96.598">
      <error>client.ClientCommunicatorException: doPost failed: Connection refused
	at client.communication.ClientCommunicator.doPost(ClientCommunicator.java:174)
	at client.communication.ClientCommunicator.submitBatch(ClientCommunicator.java:58)
	at server.ServerUnitTests.submitBatchTest(ServerUnitTests.java:151)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:31)
	at org.junit.runners.BlockJUnit4ClassRunner.runNotIgnored(BlockJUnit4ClassRunner.java:79)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:71)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:49)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:193)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:52)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:191)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:42)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:184)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:236)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:579)
	at java.net.Socket.connect(Socket.java:528)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.parseHTTPHeader(HttpClient.java:763)
	at sun.net.www.http.HttpClient.parseHTTP(HttpClient.java:633)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1323)
	at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:468)
	at client.communication.ClientCommunicator.doPost(ClientCommunicator.java:165)
	... 28 more

</error>
    </testcase>
  </testsuite>
</testrun>
